hooks/useOpenNow.tsx

```diff
@@ -0,0 +1,20 @@
+const weekday = new Array(7)
```

Comment:
this isn't really a hook, as it doesn't deal with state at all. also, this is only being used in one place correct? i don't think it needs to be its own file